lib/browser/api/browser-window.ts

```diff
@@ -91,6 +91,8 @@ BrowserWindow.fromWebContents = (webContents: WebContents) => {
 };
 
 BrowserWindow.fromBrowserView = (browserView: BrowserView) => {
+  const { webContents } = browserView;
+  if (!webContents) return null;
```

Comment:
you'll need to update the docs for `BrowserWindow.fromBrowserView` to state that it can return `null`, too.